workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-help-and-support/ddm-template.ftl

```diff
@@ -86,19 +85,19 @@
 		/>
 	</#if>
 
-	<#if supportUrlFiltered?has_content>
-		<#assign supportUrl = supportUrlFiltered[0].value?trim?replace(' ', '') />
+	<#if publisherUrlFiltered?has_content>
+		<#assign publisherUrl = publisherUrlFiltered[0].value?trim?replace(' ', '') />
 
-		<#if supportUrl?starts_with("http://") || supportUrl?starts_with("https://")>
-			<#assign sanitizedUrl = supportUrl />
+		<#if publisherUrl?starts_with("http://") || publisherUrl?starts_with("https://")>
+			<#assign sanitizedUrl = publisherUrl />
 		<#else>
-			<#assign sanitizedUrl = "https://" + supportUrl />
+			<#assign sanitizedUrl = "https://" + publisherUrl />
 		</#if>
 	</#if>
 
-	<#if supporPhoneFiltered?has_content>
+	<#if supportPhoneFiltered?has_content>
 		<#assign
-			supporPhone = supporPhoneFiltered[0].value
+			supporPhone = supportPhoneFiltered[0].value
```

Comment:
Very good, could you fix this error that was already present in the code?

```suggestion
			supportPhone = supportPhoneFiltered[0].value
```

Reply:
Yes, of course, I had mapped that out and just forgot to adjust it.
thanks for reminding me